key-uglifier/src/uglifier.rs

```diff
@@ -0,0 +1,107 @@
+use colored::Colorize;
+use hex::ToHex;
+
+fn param_set_default_encryption_key(key_length: u8) -> Vec<u8>
+{
+    let default_key = vec![0x62u8; key_length as usize];
+    return default_key
```

Comment:
In rust almost everything is an expression, so often `return` is not needed.
```suggestion
    vec![0x62; key_length as usize]
```